app/jobs/track_agent_statuses_job.rb

```diff
@@ -4,6 +4,7 @@ class TrackAgentStatusesJob
 
   def perform
     current = BackendService.new.get_agent_online_state.map do |data|
+      agent_has_eaten(data) if data[:status] == 'Ruokatunti'
```

Comment:
I think this location is kinda confusing. Why not loop over the `AgentStatus` objects later in a separate step?